assets/course-theme/contact-teacher.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @module ContactTeacher
+ * @description Responsible for making a seamless ajax post of the
+ * contact teacher form, without refreshing the whole page.
+ */
+
+/**
+ * Handles the contact teacher submit event.
+ *
+ * @param {Object} ev The contact teacher form submit event.
+ */
+function handleSubmit( ev ) {
```

Comment:
I think we can improve reaching this function.
-  If we expose the handler as something like `window.sensei.submitContactTeacher`, the form can just have an `onsubmit="sensei.submitContactTeacher(event)"` attribute, without having to find the elements and add multiple event listeners in line 74-96
- Form data can be accessed in the submit event via `event.target.element[fieldName]` ([HTMLFormElement.elements](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/elements))
- Updating state for the submit button, success feedback: would be nice if the handler could be more decoupled from the form cosmetics, maybe just toggling `is-in-progress`, `is-success` etc classes on the form element instead? Probably still needs to disable the button directly.

Reply:
Good thinking! Updated. 👍  d8d341102c342585acd089aa017b809e3b5fc82f